filehandling.c

```diff
@@ -1,12 +1,27 @@
 #include<stdio.h>
 
 struct employee
-{    char employeeName[50];    int employeeId;
+{ 
```

Comment:
First letter of  struct or function name should be capital
struct Employee

Also brackets for struct, if , for should be as
struct employee {
}
if {
}

Reply:
ok changed, please check once again